src/libraries/sendtohelix.proj

```diff
@@ -4,6 +4,9 @@
     <HelixBuild Condition="'$(HelixBuild)' == ''">$(BUILD_BUILDNUMBER)</HelixBuild>
     <HelixBuild Condition="'$(HelixBuild)' == ''">default</HelixBuild>
 
+    <!-- To enable the interpreter for mono, we need to pass an env switch -->
+    <MonoEnvOptions Condition="'$(MonoEnvOptions)' == '' and '$(MonoEnableInterpreter)' == 'true'">--interpreter</MonoEnvOptions>
```

Comment:
nit: can you inline the variable. It doesn't seem worth putting that into a separate variable.

Reply:
Ok ignore my comment, I didn't see that you are using that for different modes.